schedule/schedule-appengine/src/test/java/com/jasify/schedule/appengine/communication/CommunicatorTest.java

```diff
@@ -139,6 +139,7 @@ public void testNotifyOfInvoiceCreated() throws Exception {
         InvoicePayment payment = new InvoicePayment();
         payment.setCurrency("CHF");
         payment.setAmount(200.12);
+        payment.setFee(1.2);
```

Comment:
I did. Exception will occur. If we want null fee to work we will need some #if/#else/#end statements in the velocity template. At the moment I am expecting that fee is always set.